py-polars/src/dataframe.rs

```diff
@@ -275,9 +275,17 @@ impl PyDataFrame {
     }
 
     #[cfg(feature = "ipc")]
-    pub fn to_ipc(&self, py_f: PyObject) -> PyResult<()> {
+    pub fn to_ipc(&self, py_f: PyObject, compression: &str) -> PyResult<()> {
         let mut buf = get_file_like(py_f, true)?;
+        let compression = match compression {
+            "uncompressed" => None,
+            "lz4" => Some(IpcCompression::LZ4),
+            "zstd" => Some(IpcCompression::ZSTD),
+            s =>  panic!("compression {} not supported", s),
```

Comment:
@ritchie46 instead of panicing, shouldn't an error be returned?
```rust
            s => {
                return Err(
                    PyPolarsEr::Other(format!("compression {} not supported", s)).into(),
                )
            }
```

Reply:
Agree,  would be better

Reply:
fixed. In `to_parquet`, the same issue has occurred, which I also fixed it.